src/Components/PopupWelcome/PopupWelcome.js

```diff
@@ -1,12 +1,12 @@
 import React, { Component } from 'react';
-import styles from './WelcomePop.module.css';
+import styles from './popupWelcome.module.css';
 
-function WelcomePop(props) {
+function PopupWelcome(props) {
   return(
     <section className={styles.WelcomeSection}>
       asdasd
     </section>
   );
 }
 
-export default WelcomePop;
\ No newline at end of file
+export default PopupWelcome;
```

Comment:
вкусовщина но,
it seems to be a good practise to avoid default exports and use `named` exports https://humanwhocodes.com/blog/2019/01/stop-using-default-exports-javascript-module/

Reply:
Done
33c2ec2